eclair-core/src/main/scala/fr/acinq/eclair/channel/Helpers.scala

```diff
@@ -835,10 +836,10 @@ object Helpers {
         // if an outgoing htlc is in the remote commitment, then:
         // - either it is in the local commitment (it was never fulfilled)
         // - or we have already received the fulfill and forwarded it upstream
-        val outgoingHtlcs = localCommit.spec.htlcs.filter(_.direction == OUT).map(_.add)
-        outgoingHtlcs.collect {
-          case add if add.paymentHash == sha256(paymentPreimage) => (add, paymentPreimage)
-        }
+        localCommit.spec.htlcs
+          .collect {
```

Comment:
very nit: I'd put `.collect` on the above line

Reply:
Done with fa2a134.